index.js

```diff
@@ -0,0 +1,4 @@
+import vanilla from "./frontend-vanilla.js";
+import react from "./frontend-react.js";
+
+export { vanilla, react };
```

Comment:
```suggestion
import vanillaConfig from "./frontend-vanilla.js";
import reactConfig from "./frontend-react.js";

export { vanillaConfig, reactConfig };
```

Lite tydligare namnsättning kanske? Så slipper man döpa om i konsumerande projekt. Dvs
```diff
- import { react as caspecoReactConfig } from "@caspeco/eslint-config";
+ import { reactConfig } from "@caspeco/eslint-config";
```